misc/README

```diff
@@ -11,10 +11,10 @@ Then can add the brush to your html page:
 
 	<script type="text/javascript" src="shBrushNit.js"></script>
 
-# vim
+# Vim
 
-vim is a powerful text editor.
-The misc/vim directory contains the support of nit files to vim.
+Vim is a powerful text editor and a favorite of the Nit team.
```

Comment:
`the favorite` would probably be true, but `a favorite` makes us look like we are open to different text editors...